src/domain/reservation/manage/page/__tests__/__snapshots__/ManageReservationsPage.test.js.snap

```diff
@@ -28,7 +28,11 @@ exports[`ManageReservationsPage renders correctly 1`] = `
       </Row>
     </Grid>
     <InjectIntl(InjectT(ManageReservationsFilters))
-      filters={Object {}}
+      filters={
+        Object {
+          "date": "2019-10-14",
```

Comment:
Is this test going to fail tomorrow?

Reply:
ah true!

Reply:
fixed

Reply:
This file still needs fixing the date to mock